### ISSUE TITLE
Horizontal Server List broken in latest Vencord

### ISSUE THREAD

**Issue by NullCub3** [DiscordStyles/HorizontalServerList#44]
### Describe the bug
The Horizontal Server List is broken and does not work in the latest Vencord and Discord version

### To Reproduce  
Copy "Vencord link" into the Online Themes section
Vencord says link is valid
Server list has not changed.

### Screenshots
![themes-config](https://github.com/DiscordStyles/HorizontalServerList/assets/51034487/c6d68aa2-d08a-4c5c-916a-858befbc568b)
![server-list](https://github.com/DiscordStyles/HorizontalServerList/assets/51034487/5be623fc-17d3-4ec3-8194-d40b26f3966b)
![version](https://github.com/DiscordStyles/HorizontalServerList/assets/51034487/32a16225-23f5-47a6-822d-4ba41ee3dd6d)

### Infomation (please complete the following information)
**Discord channel**: Stable  
**OS**: Windows 10
**Mod**: Vencord
**Discord language**: English, US

### Additional context
Versions in text format are:
- Stable 240372 (63f8ac7)
- Vencord 64c6f57 (Standalone)

EDIT: Dug around in the Vencord issues, found this: 
https://github.com/Vendicated/Vencord/issues/1847

TLDR Discord updated a lot of things, which broke a lot of plugins and themes. 
The Vencord devs so far have done everything they can, so whatever is remaining is up to theme/plugin devs.

**Comment by UltLight** [comment]
I'm also having this issue. this is my version information and stuff. hopefully this helps. I love this theme 👍
![image](https://github.com/DiscordStyles/HorizontalServerList/assets/63460105/bfac2ca2-f2f9-4b52-adf8-2f0b189c1945)

**Comment by ShiiroSan** [comment]
Fixed by #45. I guess we'll have to wait until PR get merged.

**Comment by NullCub3** [comment]
Confirmed fixed with the merge of #45, thank you!
